lib/rubyhub/operations/pull_request/create.rb

```diff
@@ -17,6 +18,15 @@ def call(options)
 
           def data
             @data ||= Rubyhub::Configuration.instance.options.dig(:template, template.to_sym)
+            return unless @data.respond_to?(:to_hash)
```

Comment:
I'd suggest to make exception here to show user that nothing was initialized. Otehrwise we will have `nil` after calling `data` method because of this `return`.
(like `Rubyhub::PullRequest.new(nil).create!` above)


Please, make an exception for this case: user calls `rubyhub create` command with tag, which is not presented in yml file. It should show user, that nothing will be created, that command called with something invalid.

Reply:
This line refers to `raise IncorrectTemplateError if !template || !data || data.empty?` line. Thus, whenever `data` is `nill` it would raise an `Exception`.